### ISSUE TITLE
WPA2 Enterprise

### ISSUE THREAD

**Issue by jd4ever** [joostd/esp8266-eduroam#1]
Hi Joost, have you been able to apply your code to other WPA2 Enterprise networks?

**Comment by joostd** [comment]
Yes, there's nothing specific to eduroam. But at the moment, EAP-TLS is the only method that is supported.
